service/matching/db.go

```diff
@@ -260,12 +338,19 @@ func (db *taskQueueDB) expiryTime() *timestamppb.Timestamp {
 
 func (db *taskQueueDB) cachedQueueInfo() *persistencespb.TaskQueueInfo {
 	return &persistencespb.TaskQueueInfo{
-		NamespaceId:    db.queue.NamespaceId().String(),
-		Name:           db.queue.PersistenceName(),
-		TaskType:       db.queue.TaskType(),
-		Kind:           db.queue.Partition().Kind(),
-		AckLevel:       db.ackLevel,
-		ExpiryTime:     db.expiryTime(),
-		LastUpdateTime: timestamp.TimeNowPtrUtc(),
+		NamespaceId:             db.queue.NamespaceId().String(),
+		Name:                    db.queue.PersistenceName(),
+		TaskType:                db.queue.TaskType(),
+		Kind:                    db.queue.Partition().Kind(),
+		AckLevel:                db.ackLevel,
+		ExpiryTime:              db.expiryTime(),
+		LastUpdateTime:          timestamp.TimeNowPtrUtc(),
+		ApproximateBacklogCount: db.approximateBacklogCount.Load(),
 	}
 }
+
```

Comment:
possibly move the note below into a functional comment up here
```suggestion
emitApproximateBacklogCount gets and emits the db's approximateBacklogCount to the metrics handler. It is called after persisting the updated BacklogCount
```

Reply:
Curious - why would we move it *below* the function definition? Wouldn't that be going against the norms?